osquery/tables/system/windows/system_info.cpp

```diff
@@ -18,13 +18,16 @@
 #include "osquery/core/windows/wmi.h"
 
 namespace osquery {
+
+std::string getFqdn();
```

Comment:
Why add a function to `system.cpp` without a declaration in a header? Alternatively you can just define the function here.

For the rare cases where you don't place the declaration in the header, use `extern`.